R/return_scores.R

```diff
@@ -5,7 +5,7 @@
 #'              be manually recalculated before data analysis.
 #' @param x a \code{secuTrialdata} object
 #' @export
-#' @details return_scores will produce a data.frame that pinpoints which items are scores/calculated.
+#' @return a data.frame that pinpoints which items are scores/calculated.
```

Comment:
maybe useful to list the variables in the dataframe?

Reply:
Do you mean `name`, `itemtype` and `label`? Is this the same point as with `return_hidden_items()` because then I understand that one now.

Reply:
if that's what names(return_scores(data)) returns, yeah :)

Reply:
okedoke